embeddeddb/h2/src/main/java/org/killbill/commons/embeddeddb/h2/H2EmbeddedDB.java

```diff
@@ -146,22 +143,9 @@ public void stop() throws IOException {
         // Close the database pool ASAP to make sure the database is never re-opened by background threads
         super.stop();
 
-        try {
-            final Connection connection = DriverManager.getConnection(jdbcConnectionString, username, password);
-            if (connection != null) {
-                try {
-                    final Statement stat = connection.createStatement();
-                    // https://github.com/spotbugs/spotbugs/issues/1338 prevents us from using try-with-resource
```

Comment:
Looks like this is fixed now?

Reply:
Yes, but I think I had to make statement inside `try` scope:
```java
try (final Connection connection = DriverManager.getConnection(jdbcConnectionString, username, password);
  final Statement statement = connection.createStatement()) {
  statement.execute("SHUTDOWN");
} catch (final Exception e) {
  logger.warn("Exception while trying to shutdown H2", e);
}
```